src/libraries/Common/tests/System/Net/Http/Http3LoopbackConnection.cs

```diff
@@ -0,0 +1,126 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Net.Quic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Net.Test.Common
+{
+
+    public sealed class Http3LoopbackConnection : GenericLoopbackConnection
+    {
+        private readonly QuicConnection _connection;
+        private readonly Dictionary<int, Http3LoopbackStream> _openStreams = new Dictionary<int, Http3LoopbackStream>();
```

Comment:
```suggestion
        private readonly Dictionary<long, Http3LoopbackStream> _openStreams = new Dictionary<long, Http3LoopbackStream>();
```

Reply:
Entire test project uses int for stream IDs; don't know how much work is involved here. Leaving as TODO for post-merge.